src/modules/CountLeagues.js

```diff
@@ -0,0 +1,12 @@
+import Leagues from './leagues.js';
+
+const newLeagues = new Leagues();
+const CountLeagues = () => {
+  fetch(newLeagues.BaseUrl).then((res) => res.json())
+    .then((data) => {
+      const soccerCount = document.querySelector('.soccer-count');
+      soccerCount.innerHTML = `Soccers(${data.data.slice(0, 6).length})`;
+    });
+};
```

Comment:
CountLeagues function only returns number 6 and doesn't take any parameter here. Kindly add parameter to your function and get sum of the list items from the list container. This will help you to test your function in the next task.